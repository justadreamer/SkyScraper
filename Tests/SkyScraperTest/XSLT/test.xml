<?xml version="1.0"?>
<a>
  <c>n</c>
  <c>no</c>
  <c>nO</c>
  <c>xnO</c>
</a>
