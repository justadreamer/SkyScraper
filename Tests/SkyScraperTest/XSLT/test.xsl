<?xml version="1.0"?>
<xsl:stylesheet version="1.0" 
                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                xmlns:regexp="http://exslt.org/regular-expressions"
                extension-element-prefixes="regexp">

	<xsl:template match="c">
         regexp: no* (case)   string: <xsl:value-of select="." /> result: <xsl:value-of select="regexp:test(string(.), 'no*', 'g')" />
         regexp: no* (nocase) string: <xsl:value-of select="." /> result: <xsl:value-of select="regexp:test(string(.), 'no*', 'gi')" />
	</xsl:template>

</xsl:stylesheet>
