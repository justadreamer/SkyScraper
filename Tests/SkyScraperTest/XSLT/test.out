<?xml version="1.0"?>

  
         regexp: no+ (case)   string: n result: false
         regexp: no+ (nocase) string: n result: false
  
         regexp: no+ (case)   string: no result: true
         regexp: no+ (nocase) string: no result: true
  
         regexp: no+ (case)   string: nO result: false
         regexp: no+ (nocase) string: nO result: true
  
         regexp: no+ (case)   string: xnO result: false
         regexp: no+ (nocase) string: xnO result: true


